### PR TITLE
Update refs following go-ipfs#2812

### PR DESCRIPTION
**Pull request by RichardLitt**

### CODE REVIEW

File: apiary.apib [hunk 1]
```diff
@@ -10269,28 +10276,38 @@ Displays the hashes of all local objects.
...
-        QmNLjhs2cWVGopVTE5KqCZBdX7TE6i9AU1y8pwytt7cyzC
-        QmNLwdgyRiTuMf9M3Fjtu2TpMjz2CgZS4YgeuqjLHv3rjq
+        {
+          "Ref": "QmNLxtKFa66yoRQHWa4FXkwCMqVpWGhg2XBXAfwKvSmJFp",
+          "Err": ""
+        }
+        {
+          "Ref": "QmNXFHU3KzhzXN9pZ5Bdghk3LYGj58Xzi7WvMVVoykdLxA",
+          "Err": ""
+        }
```
Comment (Kubuxu) [inline apiary.apib:10310]: As it is NDJSON the objects should span a one line. In the response of the API it is also one line per object.

Comment (RichardLitt) [inline apiary.apib:10310]: I automatically format it to make it more readable, as all of the JSON is basically scrunched up in the terminal.

I could add a note about this to make this clear?

Comment (Kubuxu) [inline apiary.apib:10310]: Yeah, as this format is in theory incompatible with NDJSON.

Comment (RichardLitt) [inline apiary.apib:10310]: Opened #100. Can this PR be merged?

Comment (Kubuxu) [inline apiary.apib:10310]: Yup.

**Pull request merged**